ibm/service/cis/resource_ibm_cis_domain_settings.go

```diff
@@ -412,6 +413,11 @@ func ResourceIBMCISSettings() *schema.Resource {
 							Description: "security header no sniff",
 							Required:    true,
 						},
+						cisDomainSettingsSecurityHeaderPreload: {
+							Type:        schema.TypeBool,
+							Description: "security header preload",
+							Required:    true,
```

Comment:
Is this parameter required by API?
Adding a required field will break existing users.
Can't this be made optional?

Reply:
@kavya498 as per the API spec, this is a required field. I will add the respective documentation to the same.